src/core/bpf-devices.c

```diff
@@ -0,0 +1,200 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#include <linux/libbpf.h>
+
+#include "bpf-devices.h"
+#include "bpf-program.h"
+
+#define PASS_JUMP_OFF 4096
+
+static int bpf_access_type(const char *acc)
+{
```

Comment:
Please do not linebreak here. We use the following style.
```
int func(...) {
    ...
}
```

Reply:
Fixed.